### PR TITLE
Fixed variation sliders images were duplicated on product listing template.

### PR DESCRIPTION
**Pull request by bogdanarizancu**
### Ticket
- [Fix after-update issue: first preview image for variant product is twice](https://app.asana.com/0/1162879836389371/1169578275974227)

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -24,11 +24,16 @@ Gallerya transforms the WordPress native post gallery into a full fledged slides
 
 == Changelog ==
 
+= 2.2.9 =
```
Comment (juanlopez4691) [inline README.txt:27]: Please, also update the "Stable tag" entry in this file.

File: src/WooCommerce.php [hunk 2]
```diff
@@ -108,14 +108,14 @@ public static function woocommerce_template_loop_product_thumbnail() {
...
-      $attachment_ids = array_merge($attachment_ids, $wpdb->get_col($wpdb->prepare(
+      $attachment_ids = array_unique(array_merge($attachment_ids, $wpdb->get_col($wpdb->prepare(
         "SELECT pm.meta_value AS attachment_id
         FROM wp_posts p
         INNER JOIN wp_postmeta pm ON pm.post_id = p.ID AND pm.meta_key = '_thumbnail_id'
         WHERE p.ID IN ($placeholders)
         ORDER BY p.menu_order ASC",
         $variation_ids
-      )));
+      ))));
```
Comment (juanlopez4691) [inline src/WooCommerce.php:118]: Just as an idea, I wonder if we could improve the legibility of this
```php
$attachment_ids = array_unique(
  array_merge(
    $attachment_ids,
    $wpdb->get_col(
      $wpdb->prepare(
        "SELECT pm.meta_value AS attachment_id
        FROM wp_posts p
        INNER JOIN wp_postmeta pm ON pm.post_id = p.ID AND pm.meta_key = '_thumbnail_id'
        WHERE p.ID IN ($placeholders)
        ORDER BY p.menu_order ASC",
        $variation_ids
      )
    )
  )
);
```

Comment (bogdanarizancu) [reply src/WooCommerce.php:118]: Absolutely, sure!

**Comment by juanlopez4691** [comment]
@bogdanarizancu Can you please rephrase the commit message to be more concise? Perhaps something like "Fixed first slider image was duplicated for variable products." or "Fixed first product variation slider image was duplicated for variable products."?

**Pull request merged**